<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>researchRemed</class>
 <widget class="QMainWindow" name="researchRemed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно исследования препаратов</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="show">
        <property name="text">
         <string>Показать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="strategy">
        <property name="text">
         <string>Стратегия</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="spacefiller" native="true"/>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
       </property>
       <property name="cacheMode">
        <set>QGraphicsView::CacheBackground</set>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::SmartViewportUpdate</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>strategy</sender>
   <signal>clicked()</signal>
   <receiver>researchRemed</receiver>
   <slot>openStrategyMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>884</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show</sender>
   <signal>clicked()</signal>
   <receiver>researchRemed</receiver>
   <slot>openShowMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openStrategyMenu()</slot>
  <slot>openShowMenu()</slot>
 </slots>
</ui>
